cypress/e2e/AddRemoveWatchlist_spec.cy.js

```diff
@@ -0,0 +1,160 @@
+import { aliasQuery } from "../utilities/graphql-test-utils"
+
+describe("Adding/Removing from Watchlist", () => {
+  beforeEach(() => {
+    cy.intercept('POST', 'https://streamr-be.herokuapp.com/graphql', (req) => {
+      switch (req.body.operationName) {
```

Comment:
Using a switch case for the intercepts is a fantastic idea! It keeps the code DRY!